### PR TITLE
Add Minecraft and Discord group names to Groups model

### PR DESCRIPTION
**Pull request by andyksaw**
## Overview of Changes

* Adds `minecraft_name` and `discord_name` fields to the `Group` model
* This will allow PCBridge (and maybe our future Discord bot) to not need to hardcode group mappings anymore

image

## Deployment Requirements
- [ ] Composer update
- [ ] NPM update
- [ ] Database migration
- [ ] .env update
- [ ] Environment upgrade (eg. PHP version)
- [ ] Other (please specify)

### CODE REVIEW

**Comment by andyksaw** [comment]
Whoops, let me rebase that to fix the diff

File: app/Entities/Groups/Resources/GroupResource.php [hunk 1]
```diff
@@ -19,6 +19,9 @@ public function toArray($request)
             'group_id' => $this->group_id,
             'name' => $this->name,
             'alias' => $this->alias,
+            'discourse_name' => $this->discourse_name,
```
Comment (andyksaw) [inline app/Entities/Groups/Resources/GroupResource.php:22]: [review note] Probably not needed by anything consuming our API, but might as well add it for consistency

File: resources/views/admin/group/index.blade.php [hunk 2]
```diff
@@ -33,6 +35,22 @@
                             <span class="text-muted fst-italic">Unset</span>
                         @endif
                     </td>
+                    <td>
+                        @if ($group->minecraft_name != "")
+                            <i class="fas fa-cube"></i>
```
Comment (freddyheppell) [inline resources/views/admin/group/index.blade.php:40]: I feel like these icons are a bit unnecessary and add visual noise, the discourse icon is there to link to the group on the forums but I'm not sure there's anything we can link these to

Comment (andyksaw) [reply resources/views/admin/group/index.blade.php:40]: Oh right, makes sense. Let me remove those

**Review by freddyheppell — APPROVED** [review]
LGTM (other than that minor design nitpick 😅 )

**Pull request merged**